ext/objspace/objspace_dump.c

```diff
@@ -493,11 +507,12 @@ Init_objspace_dump(VALUE rb_mObjSpace)
     rb_define_module_function(rb_mObjSpace, "dump", objspace_dump, -1);
     rb_define_module_function(rb_mObjSpace, "dump_all", objspace_dump_all, -1);
 
-    sym_output = ID2SYM(rb_intern("output"));
-    sym_stdout = ID2SYM(rb_intern("stdout"));
-    sym_string = ID2SYM(rb_intern("string"));
-    sym_file   = ID2SYM(rb_intern("file"));
-    sym_full   = ID2SYM(rb_intern("full"));
```

Comment:
Rather than edit amplification, consider converting `sym_file` and `sym_full` to ragged indentation.

Reply:
Yeah, that's why I hate aligned assignation, it generates lots of diff noise. I wasn't sure what is the prefered style so I went with the existing one, but I'll follow your advice.

Reply:
I'm looking at the latest commit - did you intend to change this?

Reply:
Now yes.